public/service-worker.js

```diff
@@ -104,38 +102,53 @@ define("./service-worker.js",['./workbox-e536220e'], function (workbox) { 'use s
    * See https://goo.gl/S9QRab
    */
 
-  workbox.precacheAndRoute([{
-    "url": "/_next/static/runtime/amp.js",
-    "revision": "448564b2ecdefde43bd36b481b74be5f"
-  }, {
-    "url": "/_next/static/runtime/amp.js.map",
-    "revision": "151ff324f19c73a7d9934f1e640c6656"
-  }, {
-    "url": "/_next/static/runtime/main.js",
-    "revision": "fea8f8ade32915207a95e8f2e602c5ce"
-  }, {
-    "url": "/_next/static/runtime/main.js.map",
-    "revision": "2a4ffc62b29c786a35c62f28421cfdfb"
-  }, {
-    "url": "/_next/static/runtime/polyfills.js",
-    "revision": "cf6f4b12f4634f8f79378d41f3a855a4"
-  }, {
-    "url": "/_next/static/runtime/polyfills.js.map",
-    "revision": "82dca635a629d8ab38c3ad85b2ad65a2"
-  }, {
-    "url": "/_next/static/runtime/webpack.js",
-    "revision": "3d406d49f9f25eb6a505df647f073679"
-  }, {
-    "url": "/_next/static/runtime/webpack.js.map",
-    "revision": "1cbb5424a843655e7cfac1e6c3ba0c94"
-  }], {
-    "ignoreURLParametersMatching": [/ts/]
-  });
+  workbox.precacheAndRoute(
+    [
+      {
```

Comment:
Can you remove these files from the changes, these are auto-generated files generated by [next-pwa](https://github.com/shadowwalker/next-pwa), we might have missed git ignoring them

Reply:
Sure will do ASAP.